doc/Tex/RAM_SCB_swmf.tex

```diff
@@ -1,22 +1,13 @@
-RAM-SCB may be used as a component to the Space Weather Modeling Framework.
-Under this mode, RAM-SCB receives initial and boundary conditions from the
-Framework's other components and returns plasma properties to create a two-way
-coupled system.
+RAM-SCB may be used as a component to the Space Weather Modeling Framework. Under this mode, RAM-SCB receives initial and boundary conditions from the Framework's other components and returns plasma properties to create a two-way coupled system.
 
 %INSERT TABLE OF COUPLING DESCRIPTIONS.
 
 \section{Installing RAM-SCB as a SWMF Component}
+% QUESTION: Should this entire section just be rewritten? SWMF no longer uses CVS, we have changed the coupling on our end a bit. And SWMF has changed the coupling on their end a lot. Honestly, it may be of more use to have an entirely seperate document for the SWMF coupling as it has its own set of installation instructions and its own set of difficulties.

```

Comment:
Yes, should be rewritten. I'd start with the assumption of having either a tarball from the CSEM webpage or a checked-out copy of the repo. A new chapter is probably a good place for it, and minimal but correct is better than verbose and incorrect/outdated.

Reply:
I've left old content and our notes as comments, with a placeholder directing the user to seek assistance. Marking as resolved for now, this can be fleshed out in the near future - I'm mostly looking to get something not-out-of-date committed as a new baseline.